install/sql/adaptation.sql.template

```diff
@@ -0,0 +1,8 @@
+INSERT INTO cws_code (code)
+VALUES (
+'
+__CUSTOM_METHODS_JAVA__
+'
+);
+
+ALTER TABLE cws_worker ADD asg_class VARCHAR(10);
```

Comment:
@voxparcxls what is this asg_class?  is this m20 specific?  maybe we should change that?

Reply:
IDS maintains its own `adaptation.sql.template` file in our adaptation repo. It includes this line. I don't know that you need to add it here, too.